dejan-babic/dtPicker
# Feature login functionality

#### What's this PR do?

Adds the Login Functionality
#### Where should the reviewer start?

from the login page
#### How should this be manually tested?

hitting the login button with an empty password filed, should fire an alert message saying "Please fill in a password", filling in "any" password (for now), should redirect the user to the landing page.
#### Any background context you want to provide?

nema
#### What are the relevant tickets?

nema
